examples/hello_world.js

```diff
@@ -69,14 +69,13 @@ function main() {
   }
 
   // Delete objects we created through |new|. We just do a few of them here, but you should do them all if you are not shutting down ammo.js
-  Ammo.destroy(collisionConfiguration);
-  Ammo.destroy(dispatcher);
-  Ammo.destroy(overlappingPairCache);
+  Ammo.destroy(dynamicsWorld);
```

Comment:
sure we always need to call free() in reversed order against our malloc() it is so clear that I'm not sure the special comment needed